src/ispd/arquivo/xml/utils/WrappedElement.java

```diff
@@ -13,6 +14,49 @@
 public class WrappedElement {
     private final Element element;
 
+    /**
+     * Construct a {@link TwoStageUniform} from this instance, with the values
+     * acquired from {@link #minimum()}, {@link #maximum()} and
+     * {@link #average()}.
+     *
+     * @return {@link TwoStageUniform} with a minimum, maximum, and average values
+     * as the respective tags in this instance, and {@code 0} for  probability.
+     */
+    public TwoStageUniform toTaskSizeNoProbability() {
+        return new TwoStageUniform(this.minimum(), this.average(), this.maximum(), 0);
+    }
+
+    /**
+     * Construct a {@link TwoStageUniform} from this instance, with the values
+     * acquired from {@link #minimum()} and {@link #maximum()}.
+     *
+     * @return {@link TwoStageUniform} with a minimum and maximum values as the
+     * respective tags in this instance, and default values for the fields
+     * {@link TwoStageUniform#intervalSplit()} and
+     * {@link TwoStageUniform#firstIntervalProbability()}.
+     * @see TwoStageUniform#TwoStageUniform(double, double)
+     */
+    public TwoStageUniform toTaskSizeRange() {
```

Comment:
Could be renamed to "toUniformDistribution". This name is closer to what it actually does.